arm/avhclient/aws_backend.py

```diff
@@ -703,19 +702,31 @@ def prepare(self) -> AvhBackendState:
         self._init()
         state = self.create_or_start_instance()
         if state == AvhBackendState.CREATED:
+            logging.info(f"aws:Setting up the instance...")
```

Comment:
Remove the f-string if not required.
```suggestion
            logging.info("aws:Setting up the instance...")
```